src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -1,43 +1,154 @@
+@import (reference) "mixins";
 @import "./HXElement";
-@import "elements/hx-icon";
 @import "base/links";
+@import "elements/hx-icon";
+@import "elements/hx-progress";
+@import "components/progress";
+@import (reference) "components/buttons";
 
-#hxFileTile {
-  display: flex;
-  height: 100%;
-  width: 100%;
+// TODO: double check in IE and Edge
 
-  > * {
-    padding: 0.5rem;
+:host {
+  // reset buttons
+  button {
+    background-color: transparent;
+    border: 0;
+    color: inherit;
+    cursor: pointer;
+    padding: 0;
   }
 
-  #hxContent {
-    flex-grow: 1;
-    flex-shrink: 0;
-    padding: 0.5rem 0;
+  #hxFileTile {
+    border: 2px solid transparent;
+    color: @gray-900;
+    display: flex;
+    font-size: 0.875rem; //14px
+    height: 100%;
+    width: 100%;
   }
 
-  #hxLink {
-    display: block;
+  #hxRetry {
+    &:extend(.hxBtn all);
+    &:extend(.hxBtn.hxSm all);
+    &:extend(.hxBtn.hxTertiary all);
+
+    hx-icon {
+      // FIXME: fix in SURF-1219
+      margin-left: 0.25rem !important;
```

Comment:
NOTE: discuss in bowling. Spacing spec shows 8px, but it seems too far apart from the text for tertiary buttons.